dev/source-release.sh

```diff
@@ -112,7 +112,7 @@ tarball=$tag.tar.gz
 git archive $release_hash --worktree-attributes --prefix $tag/ -o $projectdir/$tarball
 
 echo "Signing the tarball..."
-[[ -z "$keyid" ]] && keyopt="-u $keyid"
+[[ -n "$keyid" ]] && keyopt="-u $keyid"
```

Comment:
👍

Reply:
What I've done in the past is add a function that logs either way, so logs X on true and Y on false, so we'd get `No keyid is found in variable \\$keyid, skipping adding signature to tarball` in the untrue path (just some indicator it was taken).

Reply:
Yeah trick here is missing that arg still signs, just with the default key

Reply:
Oh haha you’re right. This is why I advocate for logging somewhat verbosely in shell scripts.